app/src/main/cpp/src/components/transport_manager/include/transport_manager/bluetooth_le/ble_client.h

```diff
@@ -0,0 +1,25 @@
+#ifndef SRC_COMPONENTS_TRANSPORT_MANAGER_INCLUDE_TRANSPORT_MANAGER_BLUETOOTH_LE_BLE_CLIENT_H_
+#define SRC_COMPONENTS_TRANSPORT_MANAGER_INCLUDE_TRANSPORT_MANAGER_BLUETOOTH_LE_BLE_CLIENT_H_
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <sys/un.h>
+#include <netinet/in.h>
+#include <netdb.h> 
+
+class BleClient
+{
+    int socket_id;
+    bool connected;
```

Comment:
@ychernysheva please use names according to our coding style
```suggestion
    int socket_id_;
    bool connected_;
```

Reply:
Done in 1537bc4